src/components/Nav/Navlinkcategory.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Navlinkcategory = ({ key, name }) => {
+  return <span key={key}>{name}</span>;
+};
```

Comment:
```suggestion
const Navlinkcategory = ({ name }) => {
  return <span>{name}</span>;
};
```

key는 map 메서드 사용시에 가장 최상위 부모에 적용하는 prop 이기에
자식 컴폰너트에서 사용하지 않아도 됩니다!

Reply:
반영 하였습니다.